### DISCUSSION TITLE
it shows that someone starred the same repository 2 times

### DISCUSSION THREAD

**Discussion by mystieneko** [community/community#57352] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i don't think that's supposed to be like that

![изображение](https://github.com/community/community/assets/65071577/a15988ab-e9df-4aa5-abfe-37d22f7a7a68)

**Comment by emilburganov** [comment]
This happens when you put a star, removed it, and put it again
